sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertIntoHiveTableSuite.scala

```diff
@@ -494,4 +494,34 @@ class InsertIntoHiveTableSuite extends QueryTest with TestHiveSingleton with Bef
         spark.table("t").write.insertInto(tableName)
       }
   }
+
+  test(
+    """SPARK-20594: This is a walk-around fix to resolve a Hive bug. Hive requires that the
+      |staging directory needs to avoid being deleted when users set hive.exec.stagingdir
+      |under the table directory.""".stripMargin) {
```

Comment:
Hmm, we must create table rather than simplify by `spark.range(1).write.saveAsTable("test_table")`. Thanks again. :)